Trappist/src/Promact.Trappist.Core/Controllers/HomeController.cs

```diff
@@ -15,9 +14,7 @@ public IActionResult Error()
         {
             return View();
         }
-
-        [HttpGet]
-        [AllowAnonymous]
+        
         public IActionResult Setup(string returnUrl = null)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the overloading mechanism instead of the optional parameters. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS2360)